src/test/java/HomePageTest.java

```diff
@@ -0,0 +1,72 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.Assert;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+import java.util.Random;
+import java.util.concurrent.TimeUnit;
+
+
+public class HomePageTest extends BaseTest{
+
+    @Test
+    public void titleTest() throws InterruptedException {
+        //  go to home page https://kwidos.com/
+        // verify title is equal to kwidos.
+        driver.get("https://kwidos.com/");
+        String title = driver.getTitle();
+        Assert.assertEquals(title, "Kwidos");
+
+    }
+
+    @Test
+    public void LoginWrongCredentialsTest() throws InterruptedException {
+        driver = new ChromeDriver();
+        driver.get("https://kwidos.com/auth/login");
+        driver.manage().window().maximize();
+        driver.manage().timeouts().implicitlyWait(10, TimeUnit.SECONDS);
```

Comment:
In BaseTest you have already created new webdriver instance and set up window size and implicitlyWait. Why do you do it one more time here? Remove it please